src/test/java/com/zepben/evolve/cim/iec61970/base/wires/PowerElectronicsConnectionPhaseTest.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61970.base.wires
+
+import com.zepben.testutils.junit.SystemLogExtension
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+internal class PowerElectronicsConnectionPhaseTest {
+
+    @JvmField
+    @RegisterExtension
+    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+
+    @Test
+    internal fun constructorCoverage() {
+        MatcherAssert.assertThat(PowerElectronicsConnectionPhase().mRID, Matchers.not(Matchers.equalTo("")))
```

Comment:
Can you please static import all assertThat and matchers (like not, equalTo etc) for all your tests. It will read a lot nicer.